src/components/CharacterTab.jsx

```diff
@@ -472,73 +506,112 @@ export default function CharacterTab() {
         marginBottom: 200,
       }}
     >
-      <Flex style={{ height: '100%' }}>
-        <Flex vertical gap={8} style={{ marginRight: 8 }}>
-          <div
-            id="characterGrid" className="ag-theme-balham-dark" style={{
-              ...{ display: 'block', width: 230, height: parentH - 76 },
-              ...getGridTheme(token),
-            }}
-          >
-            <AgGridReact
-              ref={characterGrid}
-
-              rowData={characterRows}
-              gridOptions={gridOptions}
-              getRowNodeId={(data) => data.id}
-
-              columnDefs={columnDefs}
-              defaultColDef={defaultColDef}
-              deltaRowDataMode={true}
-
-              headerHeight={24}
-
-              onCellClicked={cellClickedListener}
-              onCellDoubleClicked={cellDoubleClickedListener}
-              onRowDragEnd={onRowDragEnd}
-              onRowDragLeave={onRowDragLeave}
-              navigateToNextCell={navigateToNextCell}
+      <Flex vertical gap={defaultGap}>
+        <Flex gap={10} style={{ margin: 'auto' }}>
+          <Flex style={{ width: 350 }}>
+            <Input
+              size="large"
+              style={{ height: 40 }}
+              placeholder="Search character name"
+              onChange={(e) => {
+                let newFilters = Utils.clone(namefilter.current)
+                newFilters = e.target.value.toLowerCase()
+                namefilter.current = newFilters
```

Comment:
I think all this can just be simplified to:

`nameFilter.current = e.target.value.toLowerCase()`